include/pm/device_runtime.h

```diff
@@ -32,6 +32,8 @@ extern "C" {
  * The device might be asynchronously suspended if runtime PM is enabled
  * when the device is not use.
  *
+ * @note This function is *pre-kernel-ok*.
```

Comment:
We introduced a doc mechanism to list function properties:
```
@funcprops \isr_ok
```

Reply:
ok, started using it then I saw some places using exactly this string and I couldn't find something like \pre_kernel_ok. Is it valid ?

Reply:
just found the other alias.